### ISSUE TITLE
API

### ISSUE THREAD

**Issue by bharding512** [timduly4/pyglow#24]
A couple people have asked me questions like, "what variables are available?" "How do I get a profile of oxygen density?"

As far as I know, the only way of knowing what's available is looking at the code. We need an API. Maybe we have one and I'm just not sure where it is?

**Comment by timduly4** [comment]
Good point-- are you aware of any documentation generation software?  One possibility is [doxygen](https://en.wikipedia.org/wiki/Doxygen), but there are others.

**Comment by jswoboda** [comment]
I've started to use [Sphinx](https://en.wikipedia.org/wiki/Sphinx_(documentation_generator)) and I've had some positive results. I used it to create a github page for one of my repositories which you can get from the [repository readme](https://github.com/jswoboda/PythonISRUtilities). I modified a few scripts I found to automate the process of making the github page.

**Comment by sigvaldm** [comment]
I second this. I use Doxygen for C and C++ projects, but my experience is that for Python projects, Sphinx along with ReStructured Text (reST) and [Read The Docs](https://readthedocs.org) is the way to go. Not that it is perfect, but it is well supported for these kinds of projects. It is also pretty nice to get the documentation online with Read The Docs. It also has a quite sane [Getting Started](https://docs.readthedocs.io/en/stable/intro/getting-started-with-sphinx.html) tutorial on Read The Docs.
